### ISSUE TITLE
Face Recognition using Triplet Loss Function

### ISSUE THREAD

**Issue by NEERAJAP2001** [akshitagupta15june/Face-X#840]
Here we are using OpenFace pre-trained model for facial recognition. Without going into much details on how this neural network identify two same faces.
